measurement/plugins/netflix_fast_test/measurements.py

```diff
@@ -0,0 +1,118 @@
+import requests
```

Comment:
No need for the word `test` in this module name.

This isn't a "test". It's a measurement and that's implied by the path and the package name.

`netflix_fast` is enough.